lib/acts_as_list/active_record/acts/sequential_updates_method_definer.rb

```diff
@@ -0,0 +1,23 @@
+module ActiveRecord::Acts::List::SequentialUpdatesDefiner #:nodoc:
+  def self.call(caller_class, column, sequential_updates_option)
+    caller_class.class_eval do
+      @sequential_updates = nil
```

Comment:
It seems unintuitive to set a variable at this point. Perhaps set it within `def sequential_updates?` like so:

```ruby
@sequential_updates = nil unless defined?(@sequential_updates)
```